docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1839,6 +1800,8 @@ For example, if the request comes into the Gateway over `HTTPS` but the `Service
 The opposite situation can also apply.
 However, if `GATEWAY_SCHEME_PREFIX_ATTR` is specified for the route in the Gateway configuration, the prefix is stripped and the resulting scheme from the route URL overrides the `ServiceInstance` configuration.
 
+TIP: Gateway supports all the LoadBalancer features. You can read more about them in the https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#spring-cloud-loadbalancer[Spring Cloud Commons documentation].
```

Comment:
Does this hold for `ReactiveLoadBalancerClientFilter`?

Reply:
yes